src/Catty/ViewController/Download/StoreProjectDownloader.swift

```diff
@@ -23,7 +23,7 @@
 protocol StoreProjectDownloaderProtocol {
     func fetchProjects(forType: ProjectType, offset: Int, completion: @escaping (StoreProjectCollection.StoreProjectCollectionText?, StoreProjectDownloaderError?) -> Void)
     func fetchSearchQuery(searchTerm: String, completion: @escaping (StoreProjectCollection.StoreProjectCollectionNumber?, StoreProjectDownloaderError?) -> Void)
-    func fetchProjectDetails(for project: StoreProject, completion: @escaping (StoreProject?, StoreProjectDownloaderError?) -> Void)
+    func fetchProjectDetails(projectId: String, completion: @escaping (StoreProject?, StoreProjectDownloaderError?) -> Void)
```

Comment:
One minor suggestion: could you please re-introduce the "for" -- this makes the Swift code more readable like a natural language.

Reply:
May I suggest "forId" as argument label? It would keep the nice sentence like function signature while still making it clear that the ID is in fact the argument.
(just a suggestion tho)